tc/sgx/trusted_worker_manager/enclave/enclave_data.cpp

```diff
@@ -79,7 +79,7 @@ EnclaveData::EnclaveData(const uint8_t* inSealedData) {
         nullptr, 0, &decrypted_data[0], &decrypted_size);
     tcf::error::ThrowSgxError(ret, "Failed to unseal  data");
     std::string decrypted_data_string(reinterpret_cast<const char*>(&decrypted_data[0]));
-    DeserializeSealedData(decrypted_data_string);
+    DeserializePrivateData(decrypted_data_string);
```

Comment:
Any reason to rename the function?

Reply:
Yes it is essentially deserializing only the private data.